volatility3/framework/symbols/windows/pdbutil.py

```diff
@@ -278,10 +279,15 @@ def pdbname_scan(cls,
                                                               sections = [(start, end - start)]):
             mz_offset = None
             sig_pfn = signature_offset // page_size
+            current_invalid_counter = 0
 
             for i in range(sig_pfn, min_pfn, -1):
-                if not ctx.layers[layer_name].is_valid(i * page_size, 2):
+                if current_invalid_counter > maximum_invalid_count:
```

Comment:
By not bombing out early, this may increase how long is spent on each pdb file, multiplied by the number of possible pdb files the scanner finds.  That should be limited based on the `pdb_names`, but we'll need to consider it if we ever scan for all pdb files for some reason.

Reply:
I agree